njord/tests/sqlite/open_test.rs

```diff
@@ -0,0 +1,52 @@
+use njord::keys::{AutoIncrementPrimaryKey, PrimaryKey};
+use njord::sqlite::{self};
+use njord::table::Table;
+use njord_derive::Table;
+use std::path::Path;
+
+#[derive(Table, Clone)]
```

Comment:
We dont need the structs here. Also maybe name this to "base" or "core" or something in that manner just so we can have other tests that does not apply in the others.

Reply:
Actioned ✅